pages/stammbaum.xml

```diff
@@ -1,4 +1,4 @@
-<?xml-stylesheet type="text/xsl" href="stammbaum2.xsl"?>
+<?xml-stylesheet type="text/xsl" href="stammbaum.xsl"?>
```

Comment:
Kann gelassen werden jedoch sollte dann der Name der stammbaum.xsl und stammbaum2.xsl getauscht werden, da sich die notwendigen Inhalte in stammbaum2.xsl befinden